<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessWidget</class>
 <widget class="QWidget" name="ProcessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>970</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>970</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14px;
font-weight:400;
font: url(:/text/AAvante.woff);
background-color: rgb(249, 249, 249);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="display_groupbox">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>320</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Считанные значения счётчиков и параметров</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="cycles_layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Общее количество циклов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cycles_label">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:  rgb(0, 138, 153);</string>
          </property>
          <property name="text">
           <string>4 999 999 999</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="time_layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Общее время работы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_label">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:  rgb(0, 138, 153);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>10</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="cpm_layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Граница тока насоса</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cpm_label">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:  rgb(0, 138, 153);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tp_layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Время теста насоса</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tp_label">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:  rgb(0, 138, 153);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tbc_layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Время паузы между циклами</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tbc_label">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:  rgb(0, 138, 153);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tbtp_layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Время между тестами насоса</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tbtp_label">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:  rgb(0, 138, 153);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ct__layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Время цикла установлено</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ct_label">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:  rgb(0, 138, 153);</string>
          </property>
          <property name="text">
           <string>программно</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tw_layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Время цикла</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tw_label">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:  rgb(0, 138, 153);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="edit_groupbox">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>380</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>380</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Реадктировать</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="cpm_elayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Граница тока насоса, мА</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="cpm_edit">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tp_elayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Время теста насоса [0 .. 10 000 мс]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tp_edit">
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tbc_elayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>Время паузы между циклами [0 .. 60 000], мс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tbc_edit">
          <property name="maximum">
           <number>60000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tbtp_elayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Время между тестами насоса [0 .. 28 800], с</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tbtp_edit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>28800</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ct_elayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Время цикла установлено аппаратно</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ct_edit">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tw_elayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Время цикла [40 .. 600], с</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tw_edit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="minimum">
           <number>40</number>
          </property>
          <property name="maximum">
           <number>600</number>
          </property>
          <property name="value">
           <number>40</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>52</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>52</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_write_data">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid red;
color:  rgb(0, 138, 153);
font: 15px;
font-weight:400;
background: none;
border-color: rgb(0, 138, 153);

border-width: 2px;
border-radius: 10px;
padding: 6px;</string>
           </property>
           <property name="text">
            <string>Записать параметры</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="single_cycle_groupbox">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Однократный пуск цикла</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="curr_layout">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Ток насоса во время цикла</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="curr_label">
          <property name="text">
           <string>~</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>34</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="vlt_layout">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Напряжение питания платы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vlt_label">
          <property name="text">
           <string>~</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_single_cycle">
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 1px solid red;
color:  rgb(0, 138, 153);
font: 15px;
font-weight:400;
background: none;
border-color: rgb(0, 138, 153);

border-width: 2px;
border-radius: 10px;
padding: 6px;</string>
          </property>
          <property name="text">
           <string>Тест</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="edit_counters_groupbox">
     <property name="title">
      <string>Редактировать счётчики</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="cycles_elayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Общее количество циклов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="cycles_edit">
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="time_elayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Общее время работы, c</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="time_edit">
          <property name="maximum">
           <number>2147483647</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button_write_counters">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid red;
color:  rgb(0, 138, 153);
font: 15px;
font-weight:400;
background: none;
border-color: rgb(0, 138, 153);

border-width: 2px;
border-radius: 10px;
padding: 6px;</string>
        </property>
        <property name="text">
         <string>Записать счётчики</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
